components/organisms/ToolList/nav.tsx

```diff
@@ -42,7 +42,9 @@ const Nav: React.FC<NavProps> = ({
       {toolList.map((tool, index) => {
         const pageId = filterName || insightId;
         const workspaceNavLink = `/workspaces/${workspaceId}/repository-insights/${pageId}/${tool.name.toLowerCase()}`;
-        const pageNavLink = `/${username ? `pages/${username}/` : ""}${pageId}/${tool.name.toLowerCase()}${
+        const pageNavLink = `/${
```

Comment:
@brandonroberts tagging you as I see you were in this file recently. This fixes the issue by adding `/explore/topic/`, but this link variable is doing way too much. What is the scenario this is used for when it's `pages/${username}/${pageId}/${tool.name.toLowerCase()`?

I'll probably need to add a check to only add `/explore/topic/` if there is no username.

Reply:
I broke it up in ChatGPT to see it more clearly.

```typescript
let pageNavLink = "/";

if (username) {
    pageNavLink += `pages/${username}/explore/topic/`;
} else {
    pageNavLink += "explore/topic/";
}
pageNavLink += `${pageId}/${tool.name.toLowerCase()}`;

if (selectedFilter) {
    if (Array.isArray(selectedFilter)) {
        pageNavLink += `/filter/${selectedFilter.join("/")}?range=${range ?? "30"}`;
    } else {
        pageNavLink += `/filter/${selectedFilter}?range=${range ?? "30"}`;
    }
} else {
    pageNavLink += `?range=${range ?? "30"}`;
}
```

Reply:
Not for today, but this whole tools thing probably needs a rework.